### PR TITLE
Reports List Page

### PR DESCRIPTION
**Pull request by onkar-saeloun**
## Notion card
https://www.notion.so/saeloun/Create-the-Reports-List-page-b36ccfd750af40f39864b342cdbc2ca9

## Summary

Created a page to show list of projects.

## Preview

https://user-images.githubusercontent.com/104751221/176164798-cd790f5a-202f-40e5-975e-b93a13f05cce.mp4

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Tested Manually

### Checklist:

- [ ] I have manually tested all workflows
- [ ] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

File: app/javascript/src/components/Reports/index.tsx [hunk 1]
```diff
@@ -1,126 +1,18 @@
-import React, { useState, useEffect } from "react";
```
Comment (onkar-saeloun) [inline app/javascript/src/components/Reports/index.tsx:1]: removed this page, added a separate time entry report page and added routing in same page.

**Comment by onkar-saeloun** [comment]
> @onkar-saeloun can we hide the rest of the empty reports

Ohh like for now we will show only Time entry report on list?
@supriya3105 your view?

**Comment by supriya3105** [comment]
@onkar-saeloun .  Please hide the rest of the reports for now.

**Comment by supriya3105** [comment]
@akhilgkrishnan Please review this PR again for feedback that has been implemented

File: app/javascript/src/components/Reports/timeEntry/index.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  });
+  const [selectedFilter, setSelectedFilter] = useState(filterIntialValues);
+  const [isFilterVisible, setFilterVisibilty] = useState<boolean>(false);
+  const [showNavFilters, setNavFilters] = useState<boolean>(false);
+  const [filterCounter, setFilterCounter] = useState(0);
+
+  useEffect(() => {
+    sendGAPageView();
+    setAuthHeaders();
+    registerIntercepts();
```
Comment (akhilgkrishnan) [inline app/javascript/src/components/Reports/timeEntry/index.tsx:38]: move these to `app/javascript/src/components/Reports/index.tsx`

**Review by harshaanNihal — APPROVED** [review]
LGTM

**Pull request merged**